Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM node:14.15.5-alpine3.13 as js-builder
+FROM node:15.11.0-alpine3.13 as js-builder
```

Comment:
@mckn Needed to update node alpine image - adding git didn't work for the previous image at all. Any potential issues you can think of?

Reply:
hmm, can you try the latest `node:14` image? Because node 15 is not LTS so it might be better to try to use the latest version of node 14 if that would work.

Reply:
I tried `14.16.0`(the one I updated to yesterday when I was updating Go version) but didn't work either.

Reply:
Ok ok, lets continue with this version then.